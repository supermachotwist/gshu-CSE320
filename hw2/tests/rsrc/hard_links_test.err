build list...sort...scan1...scan2...done
