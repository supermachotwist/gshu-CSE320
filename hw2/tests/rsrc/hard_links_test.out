

Hard link summary:


FILE: tests/rsrc/test_tree/file2
LINK: tests/rsrc/test_tree/file2.lnk

FILE: tests/rsrc/test_tree/file1
LINK: tests/rsrc/test_tree/file1.lnk


List of files with duplicate contents (includes hard links)

FILE: tests/rsrc/test_tree/file2
DUP:  tests/rsrc/test_tree/file2.lnk

FILE: tests/rsrc/test_tree/file1
DUP:  tests/rsrc/test_tree/file1.dup
DUP:  tests/rsrc/test_tree/file1.lnk
